.gitignore

```diff
@@ -92,7 +92,13 @@ missing
 Output.osm
 /output.osm
 /output.pgm
-plugins/*_schema.js
```

Comment:
Needed to breakout the individual schema files. The wildcard ignored the dnc schema